.circleci/config.yml

```diff
@@ -245,6 +245,9 @@ build_and_deploy: &build_and_deploy
           echo "Rebuilding cache..."
           terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr
 
+          # Post our custom Solr schema to Pantheon's Solr server
```

Comment:
@kxwang 	We should add this to our PGOV builds too.

Reply:
Great idea! Will add a story